packages/dappmanager/src/modules/stakerConfig/utils.ts

```diff
@@ -95,7 +95,8 @@ export function getNetworkStakerPkgs(network: Network): {
           {
             dnpName: "goerli-nethermind.dnp.dappnode.eth",
             minVersion: "1.0.1"
-          }
+          },
+          { dnpName: "besu-goerli.dnp.dappnode.eth", minVersion: "0.1.0" }
```

Comment:
Could you change the dnp name and repo to be consistent with the naming of the other goerli exec clients (goerli-geth and goerli-erigon) from `besu-goerli` to `goerli-besu`

Reply:
Yes will do.

Reply:
@Nabsku can yiu do this to the package and the 5 CLs? I'll fix here. And we'll pick up in a few more hours